src/main/java/com/algorand/algosdk/v2/client/algod/TransactionParams.java

```diff
@@ -0,0 +1,50 @@
+package com.algorand.algosdk.v2.client.algod;
+
+import java.io.IOException;
+
+import com.algorand.algosdk.v2.client.connect.Client;
+import com.algorand.algosdk.v2.client.connect.Query;
+import com.algorand.algosdk.v2.client.connect.QueryData;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.algorand.algosdk.v2.client.model.TransactionParametersResponse;
+
+
+/**
+ * /v2/transactions/params 
+ */
+public class TransactionParams extends Query {
+
+
+	public TransactionParams(Client client) {
+		super(client);
+	}
+
+	public TransactionParametersResponse lookup() {
```

Comment:
Lookup can be in the parent class, these all seem to be the same. Make the parent generic and pass in the response type.

Reply:
Partially addressed.